mifospay/src/main/res/values/strings.xml

```diff
@@ -248,5 +248,16 @@
     <string name="grip">Grip</string>
     <string name="hide_balance">Hide Balance</string>
     <string name="no_bank_account">No bank found</string>
+    <string name="language_type">Choose your language</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Delete this string

Reply:
It got when I created the fragment by default

Reply:
If there is no need for this string then please remove it.

Reply:
I will remove it :)